tests/extensions/tabular/test_mat_tools.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from mfr.extensions.tabular.libs import mat_h5py_scipy
+
+BASE = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestTabularMatTools:
+
+    def test_mat_73(self):
+        with open(os.path.join(BASE, 'files', 'testVer73.mat')) as fp:
+            # difference is with actual file, not function
```

Comment:
I don't think this comment is necessary since the function name `mat_h5py_scipy()` is good enough. (2 occurrences)

Reply:
I think if someone was looking at this and didn't know how it worked, the comment could be helpful. Maybe I should elaborate on it a bit more?

Reply:
Sure, let's elaborate it. (I was confused the first time when I read it, until I see the function name lol.)